hooks/13_LoginForm/Readme.md

```diff
@@ -111,7 +111,13 @@ _./src/index.html_
 - Let's build a proper _login_ layout, _loginPage.tsx_. To build a nice layout, we will install _@material-ui/core_ and _@material-ui/icons_
 
 ```bash
-npm install @material-ui/core @material-ui/icons --save
+npm install @material-ui/core@^4.0.1 @material-ui/icons@^4.0.1 --save
+```
+
+- However, we must be careful with the compatibility of certain versions of _typescript_ with the new _hooks_ of _material-ui_. For this example, we can install _typescript@3.3.3_ version.
+
+```bash
+npm install typescript@3.3.3 --save
```

Comment:
Mmm.. why we need this exact version? Do we need a latest version?

Reply:
The typescript latest versions of mark a TSERROR when using useStyles(), requiring at least one parameter but it is really optional. I had to donwgrade my workspace version of typescript and use the project's version to solve it.

Reply:
Good catch ! same happened with @Nasdan today, from my side I just added the props and good to go :), maybe we can just add the latest versions use the props as param but warn the reader that is something buggy introduced in latest ts versions that likely will be solved (we can point to the github isse)

Reply:
I have updated "Readme" and "package.json" files.